pm_icecon/bt/params/amsr2.py

```diff
@@ -35,7 +36,9 @@
             start_month=11,
             end_month=4,
             weather_filter_params=WeatherFilterParams(
-                wintrc=84.73, wslope=0.5352, wxlimt=13.7
+                wintrc=84.73,
+                wslope=0.5352,
+                wxlimt=13.7,
             ),
```

Comment:
Lol.  I tried to have these on separate lines, but one of the linters -- I thought it was black? -- put them on the same line....

Reply:
Adding a trailing `,` to the last kwarg tells black to format this way. Not really a big deal, but I like having each kwarg on it's on line :)